src/AdsImporter/Database/MongoMigration.php

```diff
@@ -173,8 +174,20 @@ public function addTransaction(ArrayableInterface $transaction): void
     {
         $document = $transaction->toArray();
 
+        if ($transaction instanceof LogAccountTransaction) {
+            $document['networkAccount']['localChange'] = $this->createMongoDate(
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "localChange" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "networkAccount" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)